app/src/main/shared/settings/default.ts

```diff
@@ -13,6 +13,9 @@ export const settings: Settings = {
         //is touchstone id NOT a version of any name in array of non-stochastic touchstones
         return !this.nonStochasticTouchstones.some((ts: string) => touchstoneId.indexOf(ts) === 0);
     },
+    hideWideFormat: function (disease) {
+        return ["MenA", "JE", "YF", "HPV", "measles", "rubella"].indexOf(disease) > -1
```

Comment:
I think Rubella and Measles are in the database with first letter upper-cased

Reply:
oh thanks for checking, in fact lets make these case insensitive for robustness